README.md

```diff
@@ -1,87 +1,190 @@
+
 A universal mechanism for calculating the estimated value of a collection of assets. This project is based on the [Card Ladder Value](https://drive.google.com/file/d/1rOY3tagsT7axRRxZWECh-0zWoMbaYbNp/view) algorithm and the [Index Mathmatics Methodology](https://www.spglobal.com/spdji/en/documents/methodologies/methodology-index-math.pdf) published by S&P Dow Jones Indices, a division of S&P Global.
 
+## Usage
+
+```tsx
+import {tami} from '@mimicry/tami';
+
+const transactions = [
+  { itemId: 'Lavender', timestamp: 1593129600000, price: 500 },
+  { itemId: 'Hyacinth', timestamp: 1600992000000, price: 700 },
+  { itemId: 'Hyacinth', timestamp: 1614211200000, price: 400 },
+  { itemId: 'Mars', timestamp: 1624406400000, price: 612 },
+  { itemId: 'Mars', timestamp: 1639008000000, price: 1200 },
+];
+
+const timeAdjustedMarketValue = tami(transactions); // 2276.3888888888887
+```
+
 ## Motivation
+
 Prior to the development of this tool, there was no standardized way to transparently value a collection of NFTs. Various data providers such as [ArtCentral](https://artcentral.io), [UpShot](https://upshot.xyz/), [bitsCrunch](https://bitscrunch.com/), [NFTGo](https://nftgo.io/), [Gallop](https://www.higallop.com/), [Banksea](https://banksea.finance/), and others have developed their own proprietary market cap methodology. This fragmentation serves some users, like traders seeking alpha, but it does not serve the best interests of all users. For example, opaque AI solutions are not sufficient for users that wish to rely upon a trustless reference price for NFTfi lending and derivative instruments. Moreover, market caps calculated using the floor price of a collection are too easily manipulated. This is especially dangerous when considered in conjunction with derivative protocols for NFT collections.
 
 **Speical Note:** _While this tool was created with NFT collections in mind, it will work for any generic basket of non-fungible assets such as trading cards, houses, music licenses, etc._
 
 ## How it Works
+
 ### Part 1. Calculate the Index Price
+
 #### Methodology
-We first calculate an "index price" for an NFT collection, calculated as follows. <br>
-```Index Price = S / (N * D)```
 
-Where: <br> 
-- ```S``` = Sum of the Last Sold Value of Every Non-Excluded NFT in Collection X
-- ```N``` = Number of Non-Excluded NFTs in Collection X with a Secondary Market Sale
-- ```D``` = Divisor<sub>old</sub> * (Index Price<sub>new</sub> / Index Price<sub>old</sub>)
+We first calculate an **Index Price** for an NFT collection, calculated as follows:
 
-For the purposes of normalization, assume that each collection's Index Price begins with a Divisor value of ```First Secondary-Market Sale Price of any NFT in the Collection / $1,000```. When the second NFT in the Index Price records its first secondary market sale, the Divisor is adjusted. The same happens when the third NFT in the Index Price records its first secondary market sale, and the fourth, and so on.
```

Comment:
@AslanGoldenhour The `createIndexValueHistory` currently initializes the `divisor` to 1 for the first transaction in a collection. This is per the white paper, and I think it makes sense to keep that as the initial value, mostly because there's no real reason to use something else.

As it turns out, that number can be any positive number and the result is the same, since it's only used to absorb the price of a first sale. Since we turn things into a ratio and then multiply by the final result, it ends up evening out.

By initializing it to 1, the result of `createIndexValueHistory` looks more like what one would expect, which is nice, and using 1 seems more straightforward than indexing on `firstPrice / 1000`.

Whatcha think?